src/js/views/patients/shared/read-only_views.js

```diff
@@ -0,0 +1,97 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import 'scss/modules/sidebar.scss';
+import 'scss/domain/action-state.scss';
+import './patient-readonly.scss';
+
+const ReadOnlyStateView = View.extend({
+  tagName: 'span',
+  className() {
+    if (this.getOption('isCompact')) return 'patient-readonly--compact patient-readonly__status';
+    return 'patient-readonly w-100';
+  },
+  template: hbs`<span class="action--{{ stateOptions.color }}">{{fa stateOptions.iconType stateOptions.icon}}{{#unless isCompact}}<span class="u-margin--l-8">{{ stateName }}</span>{{/unless}}</span>{{~ remove_whitespace ~}}`,
+  templateContext() {
+    const state = this.model.getState();
+    const isCompact = this.getOption('isCompact');
+
+    return {
+      isCompact,
+      stateOptions: state.get('options'),
+      stateName: state.get('name'),
+    };
+  },
+});
+
+const ReadOnlyOwnerView = View.extend({
+  tagName: 'span',
+  className() {
+    if (this.getOption('isCompact')) return 'patient-readonly--compact patient-readonly__owner';
+    return 'patient-readonly patient-readonly__owner w-100';
+  },
+  template: hbs`{{far "circle-user" classes="u-margin--r-8"}}<span>{{ owner }}</span>`,
+  templateContext() {
+    return {
+      owner: this.model.getOwner().get('name'),
+    };
+  },
+});
+
+const ReadOnlyDueDateView = View.extend({
+  tagName: 'span',
+  className: 'patient-readonly--compact',
+  template: hbs`
+    <span{{#if isOverdue}} class="is-overdue"{{/if}}>
+      {{far "calendar-days"}}{{#if due_date}}<span class="u-margin--l-8">{{formatDateTime due_date "SHORT" inputFormat="YYYY-MM-DD"}}</span>{{/if}}
+    </span>
+  `,
+  templateContext() {
+    return {
+      isOverdue: this.model.isOverdue(),
+    };
+  },
+});
+
+const ReadOnlyDueTimeView = View.extend({
+  tagName: 'span',
+  className: 'patient-readonly--compact patient-readonly__time',
+  template: hbs`{{far "clock"}}{{#if due_time}}<span class="u-margin--l-8">{{formatDateTime due_time "LT" inputFormat="HH:mm:ss"}}</span>{{/if}}`,
+});
+
+const ReadOnlyDueDateTimeView = View.extend({
+  template: hbs`
+    <span class="patient-readonly{{#if isOverdue}} is-overdue{{/if}}">
+      {{far "calendar-days"}}{{#if due_date}}<span class="u-margin--l-8">{{formatDateTime due_date "SHORT" inputFormat="YYYY-MM-DD"}}</span>{{/if}}
+    </span>
+    <span class="patient-readonly patient-readonly__time">
+      {{far "clock"}}{{#if due_time}}<span class="u-margin--l-8">{{formatDateTime due_time "LT" inputFormat="HH:mm:ss"}}</span>{{/if}}
+    </span>
+  `,
+  templateContext() {
+    return {
+      isOverdue: this.model.isOverdue(),
+    };
+  },
+});
+
+const ReadOnlyDurationView = View.extend({
+  tagName: 'span',
+  className: 'patient-readonly w-100',
+  template: hbs`{{far "stopwatch"}}
+    {{#if duration}}
+      <span class="u-margin--l-8">{{formatMessage (intlGet "patients.shared.readOnlyViews.readOnlyDurationView.mins") min=id}}</span>
```

Comment:
Should this be `min=duration`?

It's currently showing `NaN Minutes`: [screenshot](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/4c57ed65-2afe-4f73-b78e-9f661cac61bd).

Reply:
yep.  good catch.. ran into that writing the tests